src/main/webapp/trips/tripScript.js

```diff
@@ -12,36 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * This script sets the content width of the site. (The header and auth redirect
+ * are handled in the authScript.js file.)
+ */
+
 // Triggered upon DOM load.
 $(document).ready(() => {
   // Redirect to homepage if user is not signed in.
-  isSignedIn().then((signInStatus) => {
-    if (!signInStatus) {
-      window.location.replace('/');
-    } else {
+  getAuthObject().then((authObject) => {
+    if (authObject.loggedIn) {
+      // Set the content width of the site.
       setContentWidth('800px');
-      displayHeader();
     }
   });
 });
 
-// Returns Promise with the sign in status in a boolean.
-function isSignedIn() {
+// Returns Promise with the auth object, containing login status and information.
+function getAuthObject() {
```

Comment:
this function is also in authScript.js. Is it necessary in both... (genuine question)

Reply:
That's a good point, I actually don't need this -- thanks! Removed :)